packages/flutter/test/material/dropdown_test.dart

```diff
@@ -1942,7 +1942,7 @@ void main() {
                   selectedItem = string;
                 }),
                 selectedItemBuilder: (BuildContext context) {
-                  return items.map<Widget>((String item) {
```

Comment:
Why remove the type parameter here?

Reply:
This is not the real issue situation.

Reply:
Right -- Basically, what @xu-baolin means here is that we should explicitly require `selectedItemBuilder` to return a list of `Widgets` internally. Before his change, this test would have failed if `<Widget>` was not specified. With his change, not explicitly adding the type parameter does not trigger the crash/failure.

Maybe we could add a comment above [the selectedItemBuilder code](https://github.com/flutter/flutter/blob/89a7bd05bdb7445d4815bde4300a7115a563c182/packages/flutter/lib/src/material/dropdown.dart#L1306) describing this? It might help future contributors/readers understand this requirement.